src/external/VTK.jl

```diff
@@ -0,0 +1,59 @@
+import WriteVTK: vtk_grid, vtk_save
+
+# Uses WriteVTK.jl to convert scfres structure to VTk file format
+
+@doc raw"""
+    save_scfres(filename, scfres, Val(:vtk))
```

Comment:
Not in sync with the function definition, maybe just scratch this line completely ... it tends to become out of sync super quickly.